charts/move2kube/templates/NOTES.txt

```diff
@@ -17,23 +18,38 @@ save move2kube transformation Knative function       {{ $move2kubeSaveTransforma
 {{/* Empty line */}}
 
 Run the following commands to wait until the move2kube workflow dependencies and build are done and workflow is running on namespace {{ $ns }}:
-  oc wait -n {{ $ns }} pod --for=condition=Ready=true -l app={{ .Values.instance.name }} {{ $timeout }}
+  oc wait -n {{ $ns }} pod --for=condition=Ready=true -l "app=m2k" {{ $timeout }}
   oc wait -n {{ $ns }} ksvc {{ .Values.kfunction.name }} --for=condition=Ready=true {{ $timeout }}
   oc wait -n {{ $ns }} sonataflow/{{ .Values.workflow.name }} --for=condition=Running {{ $timeout }}
 
+{{/* Empty line */}}
+To create SSH Keys secret for move2kube instance and configure SCC, run:
+oc -n {{ $ns }} adm policy add-scc-to-user anyuid -z default
+oc -n {{ $ns }} create secret generic sshkeys --from-file=id_rsa=${HOME}/.ssh/id_rsa --from-file=id_rsa.pub=${HOME}/.ssh/id_rsa.pub
+
+{{/* Empty line */}}
+Then force the pod to be recreated:
+oc -n {{ $ns }} scale deploy move2kube --replicas=0 && oc -n {{ $ns }} scale deploy move2kube --replicas=1
+
 {{/* Empty line */}}
 Once the wait commands are completed successfully, there is a need to edit the move2kube workflow with a link to the move2kube route.
-Run the following commands to complete the installation:
+As the Knative service cannot be updated, we need to delete if first and then re-create it with the helm command.
+Run the following command or follow the steps prompted at the end of the workflow installation to apply it to the move2kubeURL parameter:
 
-  M2K_ROUTE="https://$(oc -n {{ $ns }} get routes move2kube-route -o yaml | yq -r .spec.host)"
-  oc -n {{ $ns }} delete ksvc m2k-save-transformation-func && \
-    helm upgrade move2kube orchestrator-workflows/move2kube -n {{ $ns }} --set workflow.move2kubeURL=${M2K_ROUTE}
+  M2K_ROUTE=$(oc -n {{ $ns }} get routes move2kube-route -o yaml | yq -r .spec.host)
+  oc -n {{ $targetNS }} delete ksvc m2k-save-transformation-func && \
+    helm upgrade move2kube orchestrator-workflows/move2kube -n {{ $targetNS }} --set workflow.move2kubeURL=https://${M2K_ROUTE} --set brokerName={{ $brokerName }} --set brokerNamespace={{ $targetNS }}
```

Comment:
I'm fine with re-using the `targetNS` for the `brokerNamespace` here as this is for example, but in general, this may not be true and the broker can be anywhere :)